### ISSUE TITLE
Multiline custom XCTest errors are cut off in inline test output

### ISSUE THREAD

**Issue by jm-alan** [swift-server/vscode-swift#450]
**Describe the bug**
When providing a multiline string as a custom message for an XCTest assertion failure, only the first line of output is displayed.

**To Reproduce**
1. Create a simple test case
2. Provide a custom assertion message using a multiline string
3. Fail the assertion
e.g.
```swift
import XCTest

final class BuggyFailureDisplay: XCTestCase {
    func testBuggyAssertionFailure() {
        XCTFail("""
        You would expect this entire error to display,
        however, every line from this one onward will
        be cut off in the inline failure view, despite
        appearing correctly in XCode and command
        line output
        """)
    }
}
```

**Expected behavior**
The entire string is displayed

**Environment**
 - OS: macOS Monterey 12.6.1
 - Swift version
 ```
swift-driver version: 1.62.15
Apple Swift version 5.7.1 (swiftlang-5.7.1.135.3 clang-1400.0.20.51)
Target: arm64-apple-macosx12.0
```
 - Visual Studio Code version: `1.74.0`
 - vscode-swift version: `v0.9.0`

**Additional context**

![Screen Shot 2022-12-14 at 7 51 52 PM](https://user-images.githubusercontent.com/70044486/207746881-ea4b8b79-7c5c-4242-b25c-34c7f266c834.png)

**Comment by adam-fowler** [comment]
With the current output from XCTest it isn't always possible to do this and get the correct result 100% of the time. The problem occurs when general output from tests gets intermingled with the XCTest output. If you have a print statement immediately after the `XCTFail` in most cases it is not possible to differentiate between the end of the error message and the print statement. 

I will add code to support multi line errors, they just might be a little longer than expected sometimes

**Comment by adam-fowler** [comment]
Fixed in #455

**Comment by jm-alan** [comment]
> If you have a print statement immediately after the `XCTFail` in most cases it is not possible to differentiate between the end of the error message and the print statement.

I apologize for not looking too far into the code being used to parse the XCTest output, but this makes sense. Still, thank you!
